### DISCUSSION TITLE
404 errors

### DISCUSSION THREAD

**Discussion by jonyschuk** [npm/feedback#961] [Registry] [answered]
Getting 404 errors on packages that should exist.

`error An unexpected error occurred: "https://registry.yarnpkg.com/@aws-sdk/middleware-signing/-/middleware-signing-3.354.0.tgz: Request failed \"404 Not Found\""`

**Comment by Ricardonacif** [comment]
Same. On random packages.

**Reply by jonyschuk** [reply]
Yes, I'm seeing random packages as well. It looks like they are looking into it now.

https://status.npmjs.org/

**Reply by lekevicius** [reply]
Switched to https://registry.npmmirror.com/ and it works. So clearly something is wrong.

**Comment by MylesBorins** [comment, marked as answer]
We had an incident on Friday but everything is working now

https://status.npmjs.org/incidents/jglnklhspsnm
